website/project/utils.py

```diff
@@ -6,16 +6,33 @@
 
 # Alias the project serializer
 from website.project.views.node import _view_project
-serialize_node = _view_project
+serialize_node = _view_project  # TODO: Seriously?
+
+CONTENT_NODE_QUERY = (
+    # Can encompass accessible projects, registrations, or forks
+    Q('is_bookmark_collection', 'ne', True) &
```

Comment:
You can't have a bookmark collection without it also being a collection, so you can just use `is_collection` here.